src/main/java/com/example/bank/bank/transfer/transfer/history/export/XlsxTransferHistoryGenerator.java

```diff
@@ -0,0 +1,91 @@
+package com.example.bank.bank.transfer.transfer.history.export;
+
+import com.example.bank.bank.transfer.transfer.history.TransferHistory;
+import com.example.bank.bank.transfer.transfer.history.TransferHistoryService;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.apache.poi.ss.usermodel.*;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.stereotype.Service;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.List;
+
+
+@Service
+@RequiredArgsConstructor
+@Getter
+public class XlsxTransferHistoryGenerator {
+    public static final String FILE_NAME_PATTERN = "Transfer history %s.xlsx";
```

Comment:
jaki jest powod ze to musi byc public?

Reply:
Powodem jest wykorzystanie w kontrolerze, bo plik xlsx jest pobierany bezpośrednio z przeglądarki. Nie chciałem, żeby to pole było zadeklarowane w kontrolerze (treść Stringa) .